src/common/state-button.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import ActionButton from './action-button'
+import Colors from './Themes/default.js'
+import propTypes from './prop-types'
+
+const Button = styled(ActionButton)`
+  background-color: #fff
+  border: 2px solid ${props => props.enabled ? Colors.enabled : Colors.disabled}
+  color: ${props => props.enabled ? Colors.enabled : Colors.disabled}
+  font-size: 12px
```

Comment:
Avoid defining a fixed font size, you can use relative unit if you want to (`em`).